pkg/apis/kops/v1alpha2/cluster.go

```diff
@@ -665,8 +665,10 @@ type EtcdManagerSpec struct {
 	// This allows etcd setting to be configured/overwriten. No config validation is done.
 	// A list of etcd config ENV vars can be found at https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/configuration.md
 	Env []EnvVar `json:"env,omitempty"`
-	// BackupInterval which is used for backups. The default is 15 minutes.
+	// BackupInterval which is used for backups.
```

Comment:
Why is the mention of a default removed from v1alpha2 but not the internal or v1alpha3 APIs?

Reply:
Typo. Fixed.